operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -375,11 +375,20 @@ Verify that the Lustre file system is available from the management cluster.
         ```text
         cray-console-node-1      3/3     Running            0          2m
         ```
+1. Ensure that the storage nodes are already joined to spire.
 
-1. Determine whether the `cfs-state-reporter` service is failing to start on each manager/master and worker NCN while trying to contact CFS.
+    ```bash
+    /opt/cray/platform-utils/spire/fix-spire-on-storage.sh
+    ```
+
+    Note that if the script errors out indicating that a file is missing, subsequent re-runs may allow it to complete successfully.
+
+1. Determine whether the `cfs-state-reporter` service is failing to start on each manager/master, worker, and storage NCN while trying to contact CFS.
+
+    PLEASE NOTE:  In the command below, modify the range of the worker and storage nodes BASED ON THE NUMBER THAT EXIST ON THE SYSTEM.
 
     ```bash
-    ncn-m001# pdsh -w ncn-m00[1-3],ncn-w00[1-3] systemctl status cfs-state-reporter
+    ncn-m001# pdsh -w ncn-m00[1-3],ncn-w00[1-3],ncn-s00[1-3] systemctl status cfs-state-reporter
```

Comment:
The note should be removed, and this should just find the NCNs from `/etc/hosts` or from an API.

```bash
pdsh -b -w $(grep -oP 'ncn-\w\d+' /etc/hosts | sort -u |  tr -t '\n' ',') '
systemctl status cfs-state-reporter
'
```